dashboard/src/components/Tabs/Tabs.tsx

```diff
@@ -63,10 +63,12 @@ const TabsComponent = ({
       defaultValue={defaultTab}
       className="w-full"
     >
-      <TabsList className="w-full justify-start bg-transparent">
-        {tabsTrigger}
-      </TabsList>
-      <div className="border-t border-darkGray py-6">{filterListElement}</div>
+      <div className="sticky top-16 z-[5] rounded-md bg-lightGray pb-6 pt-12">
```

Comment:
try swapping rounded-md with `drop-shadow`

Reply:
`drop-shadow` gets a bit weird when nothing else in the page has shadows, but we might try something else another time